RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -2253,3 +2253,34 @@ void consumeStuff()
 		return;
 	}
 }
+
+// In standard or with few IOTMs we might not be able to fill spleen with adventures or worksheds
+// So in that case we can use them for the low priority various drops
+boolean cannotFillSpleenWithHighPriority()
```

Comment:
```suggestion
boolean shouldUseSpleenForLowPriority()
```
"cannot fill" implies to me that if we have no spleen left it should be true, but actually it's false